noxfile.py

```diff
@@ -16,7 +16,22 @@ def docs(session):
 
 @nox.session(name="docs-live")
 def docs_live(session):
-    """Build and serve the documentation with live reload."""
+    """Build a live server with MyST documentation."""
     session.install("-r", "requirements.txt")
     session.chdir("docs")
     session.run("myst", "start")
+
+
+@nox.session(name="redirects")
+def redirects(session):
+    """Generate redirects for built docs."""
+    session.run(
+        "uv",
+        "run",
+        "https://raw.githubusercontent.com/jupyter-book/jb1-redirect-generator/main/generate_redirects.py",
```

Comment:
Should we point to a specific commit here? It seems a bit iffy just pointing to the main branch and assume it will always work.

In fact, if we want docs to build alone, maybe we should just vendor this (which means we're sort of back to where we were, but this time with dev happening upstream...)

Reply:
It felt like this "run with uv" was a nice balance between "make it a pip package" and "copy paste the script everywhere"